### DISCUSSION TITLE
Shared Event Calendar

### DISCUSSION THREAD

**Discussion by xtagon** [BattlesnakeOfficial/feedback#57] [Feature Requests]
It would be nice to have a shared calendar URL (iCal, Google Calendar, or something similar) so that users could subscribe to sync events such as tournaments and Twitch shows.

**Comment by bvanvugt** [comment]
Love this, we want this internally too. Real question is how best to implement, does anyone have examples of this being done well?

**Reply by xtagon** [reply]
There's a couple of ways I can think of

1. Manual method: just create a Google Calendar and share a public iCal URL from it. People who copy the iCal URL could sync it to Google Calendar, or Outlook, or whatever they use. This is probably the simplest method.

2. Automated method: If the Battlesnake Play database already has the schedule as structured data, then it is possible to implement an iCal endpoint that exposes those dates. This sounds much more complicated, but I guess it depends on what is available in terms of libraries that already implement most of the spec for you? I am sure there are iCal serializers for Golang, not so sure how easy this would be though.

**Comment by andrewdmaclean** [comment]
@xtagon the calendar is now live at https://play.battlesnake.com/calendar

**Reply by xtagon** [reply]
> @xtagon the calendar is now live at https://play.battlesnake.com/calendar

Awesome!! It synced to Google Calendar with one click
